integration-tests/plugins/__tests__/cart/store/create-cart.spec.ts

```diff
@@ -41,6 +46,38 @@ describe("POST /store/carts", () => {
   })
 
   it("should create a cart", async () => {
+    const region = await regionModuleService.create({
+      name: "US",
+      currency_code: "usd",
+    })
+
+    const api = useApi() as any
+    const response = await api.post(`/store/carts`, {
+      email: "tony@stark.com",
+      currency_code: "usd",
+      region_id: region.id,
+    })
+
+    expect(response.status).toEqual(200)
+    expect(response.data.cart).toEqual(
+      expect.objectContaining({
+        id: response.data.cart.id,
+        currency_code: "usd",
+        email: "tony@stark.com",
+        region: expect.objectContaining({
+          id: region.id,
+          currency_code: "usd",
+        }),
+      })
+    )
+  })
+
+  it("should use any region", async () => {
```

Comment:
Is the region selection deterministic when creating the cart? If so, it would be good to create 2 regions, and assert that eg. the first one is used. If not, we can still assert that the region is present in the response.

Reply:
> If not, we can still assert that the region is present in the response.

@sradevski, can I get you to expand on this?

Reply:
Good push, this was an oversight from my side. Will add in